app/views/devise/registrations/edit.html.erb

```diff
@@ -37,7 +37,7 @@
 
         <div class="form-group">
           <%= f.label :gender %><br />
-          <%= f.select :gender, [["Male", "M"], ["Female", "F"]], { prompt: 'Select Gender' }, { class: "form-control" } %>
+          <%= f.select :gender, select_options_for_gender, gender_select_prompt, { class: "form-control" } %>
```

Comment:
Please avoid unnecessary method defining. I think we don't need `gender_select_prompt` & `select_options_for_gender ` methods here.

Reply:
Done Sir.